src/spectrecoff-cli/commands/Grid.fs

```diff
@@ -0,0 +1,78 @@
+﻿namespace SpectreCoff.Cli.Commands
+
+open Spectre.Console
+open Spectre.Console.Cli
+open SpectreCoff
+
+type GridSettings()  =
+    inherit CommandSettings()
+
+type GridExample() =
+    inherit Command<GridSettings>()
+    interface ICommandLimiter<GridSettings>
+
+    override _.Execute(_context, _settings) =
+        let numbersGrid = grid [
+            Numbers [1; 2]
+            Strings ["One"; "Two"; "Three"]
+        ]
+        Many [
+            C "The grid will have as many columns as are needed to accomodate the longest row:"
+            numbersGrid.toOutputPayload
+        ]
+        |> toConsole
+
+        addRowToGrid numbersGrid  (Numbers [3; 4; 5])
+        Many [
+            C "Rows can later be added to an existing grid. Keep in mind that the number of elements per row must not exceed the number of columns:"
+            numbersGrid.toOutputPayload
+        ]
+        |> toConsole
+
+        let renderableGrid = grid [
+            Renderables [numbersGrid; numbersGrid]
```

Comment:
It's a good question nowadays if this case is needed, as any renderable can be mapped easily to an outputpayload via the extension method. We can recap this design choice after the PR, too.